app/main.py

```diff
@@ -6,9 +6,11 @@
 
 app = FastAPI()
 
+security_token = "qwertyuiopasdfghjklzxcvbnm" 
 
 @app.get("/")
 def root():
+    print("I am here")
     return {"message": "Fast API in Python"}
```

Comment:
Storing a security token directly in the code is a security risk. It's recommended to use environment variables or a secure vault for storing sensitive information like tokens. This approach not only enhances security but also makes it easier to update the token without changing the codebase.

**Label**: security

**Code suggestion**

```python
import os

security_token = os.getenv('SECURITY_TOKEN')
```